earthaccess/api.py

```diff
@@ -197,7 +199,7 @@ def get_s3_credentials(daac: str, provider: str) -> Dict[str, Any]:
     return earthaccess.__auth__.get_s3_credentials(daac=daac, provider=provider)
 
 
-def collection_query(cloud_hosted: bool = True) -> Type[CollectionQuery]:
+def collection_query() -> Type[CollectionQuery]:
     """Returns a query builder instance for NASA collections (datasets)
 
     Parameters:
```

Comment:
suggest we remove this part of the docstring (GH won't let me do so as a suggestion)